game/racer.go

```diff
@@ -0,0 +1,10 @@
+package game
+
+import "fmt"
+
+type Racer struct {
+}
+
+func (r *Racer) Play() {
```

Comment:
you do not need a pointer on the receiver

Reply:
Pointer has been removed